com.unity.render-pipelines.universal/Editor/ScriptableRendererFeatureProvider.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    class ScriptableRendererFeatureProvider : FilterWindow.IProvider
+    {
+        class FeatureElement : FilterWindow.Element
+        {
+            public Type type;
+        }
+
+        readonly ScriptableRendererDataEditor m_Editor;
+        public Vector2 position { get; set; }
+
+        public ScriptableRendererFeatureProvider(ScriptableRendererDataEditor editor)
+        {
+            m_Editor = editor;
+        }
+
+        public void CreateComponentTree(List<FilterWindow.Element> tree)
+        {
+            tree.Add(new FilterWindow.GroupElement(0, "Renderer Features"));
+            var types = TypeCache.GetTypesDerivedFrom<ScriptableRendererFeature>();
+            var data = m_Editor.target as ScriptableRendererData;
+            foreach (var type in types)
+            {
+                if (data.DuplicateFeatureCheck(type))
+                {
+                    continue;
+                }
+
+                string path = GetMenuNameFromType(type);
+                tree.Add(new FeatureElement
+                {
+                    content = new GUIContent(path),
+                    level = 1,
+                    type = type
+                });
+            }
+        }
+
+        public bool GoToChild(FilterWindow.Element element, bool addIfComponent)
```

Comment:
Missing override keyword?

Reply:
`override` is not used when implementing interfaces